.github/workflows/deploy_branch.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     uses: zooniverse/ci-cd/.github/workflows/npm_build.yaml@main
     with:
       commit_id: ${{ github.sha }}
-      node_version: '16.13.x'
+      node_version: '16'
```

Comment:
these will need the `--legacy-peer-deps` flag added to .npmrc or we allow flags to flow through the ci/cd scripts

Reply:
thanks 👍🏼